packages/vuetify/src/components/VField/VField.sass

```diff
@@ -131,10 +131,26 @@
   padding-bottom: var(--v-field-input-padding-bottom)
   position: relative
   width: 100%
+  z-index: -1
```

Comment:
Did you check the docs to make sure this didn't cause any weird side effects?

Reply:
Did local doc testing after the latest change